### DISCUSSION TITLE
hello. I am someone who would like to purchase T41U5XBB board and try out axis control.

### DISCUSSION THREAD

**Discussion by ChoEunHyeon** [phil-barrett/grblHAL-teensy-4.x#85] [Q&A]
hello.
Thank you for selling a good product.
I want to use this board to control the axes of machines other than CNC.
I have a question.

1. Is it possible to control not only CNC but also all machines with 3, 4, and 5 axes through this?

2. I know that this control is control using G code. I would like to know if this is pid control using encoder value.

3. What should I do if I want to continuously purchase this board when building a machine?
Do you plan to continue selling for a while?
 
Could you please leave me your email address so I can ask you if I have any questions?

**Comment by phil-barrett** [comment]
Hello. Thank you for your order!  I will take it to the Post Office this morning. I have sent email to the address you used with Tindie order. Please use my address on that to reply. I am very happy to answer any questions you have. Please feel free to ask.
 
1) The T41U5XBB uses grblHAL.  The T41U5XBB is designed for grblHAL.  It only controls stepper and servo motors that use Step and Direction signals.  Spindle Motor using PWM or 0-10V is supported.

2) Servo motors are supported. Servo motors use encoders.  There is no direct support on T41U5XBB for encoder to support motion control.

3) Contact me when you wish to purchase multiple T41U5XBB. 1 - 4 boards: full price.  >4 boards: discounted price.
